client/src/components/single-dog/SingleDog.css

```diff
@@ -42,11 +47,19 @@
   margin: 10px;
 }
 
+.options-wrapper {
+  display: flex;
+  justify-self: end;
+  height: 100vh;
```

Comment:
KILL IT WITH FIRE 🔥 
What I would like to say is: There are plenty of options to give this element the right size, don't use viewport units deeper down in the DOM. They are good at higher levels to give a layout element a size, but not here.